### PR TITLE
User profile

### PR DESCRIPTION
**Pull request by sohan9819**
profile page updated 
added profile update feature

### CODE REVIEW

File: src/components/ProfileInfo.jsx [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import { useSelector } from 'react-redux';
+import { selectCurrentUser } from '../features/auth/authSlice';
+
+export const ProfileInfo = () => {
+  const user = useSelector(selectCurrentUser);
+  const { followers, following, createdAt } = user;
+
+  // let formatedDate = new Date(createdAt).toLocaleString('en-US', {
```
Comment (chanchal-panpaliya) [inline src/components/ProfileInfo.jsx:8]: remove commented code

File: src/components/ProfileForm.jsx [hunk 2]
```diff
@@ -1,75 +1,224 @@
 import { FiEdit2 } from 'react-icons/fi';
+import { useSelector } from 'react-redux';
+import {
+  selectCurrentUser,
+  selectCurrentToken,
+  updateUser,
+} from '../features/auth/authSlice';
```
Comment (chanchal-panpaliya) [inline src/components/ProfileForm.jsx:7]: You can explore absolute imports, it makes the code look cleaner
[https://dev.to/mr_frontend/absolute-imports-in-create-react-app-3ge8](url)

**Review by chanchal-panpaliya — APPROVED** [review]
1.Nice UI
2. Good Folder structure
3. Good to merge

File: src/components/ProfileForm.jsx (hunk 2 above)
Comment (sohan9819) [reply src/components/ProfileForm.jsx]: Sure , Thanks for the suggestions . Done making the suggested changes

**Review by rahul31agrawal — APPROVED** [review]
Hi Sohan
Looks good to merge

**Pull request merged**